src/core/InputDropdown/style.ts

```diff
@@ -267,7 +249,13 @@ const doNotForwardProps = ["intent", "open", "sdsStage"];
 export const StyledInputDropdown = styled(Button, {
   shouldForwardProp: (prop: string) => !doNotForwardProps.includes(prop),
 })`
-  ${labelFontBodyS}
```

Comment:
We changed the default font body size style to be size xs (`labelFontBodyXs`) because all of the pieces of the inputDropdowns are size xs, _except_ for the minimal variant label and its details. So previously the label for all the InputDropdowns was being set to override this default (which had been `labelFontBodyS`) to be size xs. This in turn prevented the minimal variant's label from being able to override this.

So now the default is xs, and it doesn't have to be overridden by the label, details, and counter below. This also _does_ allow it to be overridden by just the minimal variant's label.